backend/src/ml_space_lambda/metadata/lambda_functions.py

```diff
@@ -58,6 +58,10 @@
 
 @api_wrapper
 def compute_types(event, context):
+    return get_compute_types()
+
+
+def get_compute_types():
     response = {"InstanceTypes": {}, "AcceleratorTypes": {}}
 
     if this.cached_response_compute_types:
```

Comment:
Why are these two separate functions?

Reply:
I wanted to leave the lambda function as-is to not break any APIs, but I also now need to be able to get the compute types without dealing with the API wrapper

Reply:
can you add a comment to explain that?

Reply:
What kind of comment are you thinking? I don't like leaving comments in code like "we abstracted to a function because this logic is reused" since code comments are typically to explain what's happening, not why something was done.